wallet-webapps-reward/src/main/webapp/vue-app/components/reward/SendRewardsTab.vue

```diff
@@ -305,6 +307,8 @@ export default {
   data() {
     return {
       search: '',
+      id: `DatePicker${parseInt(new Date().toLocaleDateString())}`,
```

Comment:
This generate a date on format :

> console.log(event.toLocaleDateString(undefined, options));
// expected output: Thursday, December 20, 2012 (varies according to default locale)
I prefer that you use a timestamp with Date.now()

```suggestion
      id: `DatePicker${Date.now()}`,
```